### PR TITLE
Jjo actions

### PR DESCRIPTION
**Pull request by jjo**
Implemented 'list-br' and 'list-ports' actions wrappers on top of ovs-vsctl ...

### CODE REVIEW

**Comment by ghost** [comment]
hey bose, this looks pretty good bose!

File: actions/ovs.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    output = subprocess.check_output(cmd)
+    action_set({'output': output.strip()})
+
+
+def list_ports(args=None):
+    ''' List ports of an existing OVS bridge '''
+    bridge = action_get("bridge")
+    cmd = ["ovs-vsctl", "--", "list-ports", bridge]
+    log('Listing ports: {}'.format(cmd))
+    output = subprocess.check_output(cmd)
```
Comment (sabaini) [inline actions/ovs.py:46]: How do we do Error handling?

Comment (jjo) [reply actions/ovs.py:46]: It's handled at main() if got an Exception, see:
   action_fail("Action {} failed: {}".format(action_name, str(e)))

Comment (auria) [inline actions/ovs.py]: Exceptions can be raised via StdErr

**Review by ghost — APPROVED** [review]
Looks pretty good boss

**Pull request merged**